ReactCommon/jsi/jsi/jsi.h

```diff
@@ -148,9 +148,11 @@ class Runtime {
  public:
   virtual ~Runtime();
 
+#ifdef YI_JSCORE_AVAILABLE
   /// Returns the context. Only use if you know what you are doing.
   /// For JSCRuntime returns a JSGlobalContextRef
   virtual void *internalContext() = 0;
```

Comment:
I lied... YI_JSCORE_AVAILABLE is likely not present in this project. I think it would be better to just make it return null as a default implementation and only implmeent in the JSCruntime class. Otherwise the ReactCommon library will not have it in the future when we get to modular choosing betwen the libs

Reply:
the problem here is that we'll have to modify hermes too.
the `JSCRuntime` interface must match on hermes' side too.

Reply:
Ah, I see the whole problem now then... Can you think of another way? The true solution is to use the JSI way of replacing native functions, which means we can then eliminate the need to return the internalContext... Until then we need to choose between app side choice between executors, changing hermes too, or removing the ability to change JS time